pmd-core/src/main/java/net/sourceforge/pmd/util/ClassUtil.java

```diff
@@ -17,7 +17,9 @@
  * UIs without the package prefixes.
  *
  * @author Brian Remedios
+ * @deprecated Is internal API
  */
+@Deprecated
```

Comment:
Why not `@InternalApi`?

Reply:
It looks like this was used only by type and method properties, which are deprecated for removal.